src/setup/devnet/index.js

```diff
@@ -1505,7 +1506,7 @@ export class Devnet {
                 ),
                 execa(
                   'jq',
-                  ['\'.config.consensus = "bor" | .config.eip1559FeeCollectorTransition = .config.londonBlock | .config.eip1559FeeCollector = .config.bor.burntContract."0"\'', `${this.erigonGenesisFilePath(i)} > ~/tmp.json && mv ~/tmp.json ${this.erigonGenesisFilePath(i)}`],
+                  ['\'.config.consensus = "bor" | .config.bor.agraBlock = .config.shanghaiBlock | del(.config.shanghaiBlock)\'', `${this.erigonGenesisFilePath(i)} > ~/tmp.json && mv ~/tmp.json ${this.erigonGenesisFilePath(i)}`],
```

Comment:
what does `del` do?

Reply:
deletes, we don't want a shanghaiBlock in case of erigon